src/Css/Browser.Css.Ext.fs

```diff
@@ -8,6 +8,24 @@ type Window with
     [<Emit("$0.getComputedStyle($1...)")>]
     member __.getComputedStyle(elt: Element, ?pseudoElt: string): CSSStyleDeclaration = jsNative
 
+type Element with 
+    /// returns this DocumentOrShadow adopted stylesheets or sets them.
+    /// https://wicg.github.io/construct-stylesheets/#using-constructed-stylesheets
+    [<Emit("$0.adoptedStyleSheets{{=$1}}")>]
+    member __.adoptedStyleSheets with get(): CSSStyleSheet array = jsNative and set(v: CSSStyleSheet array) = jsNative
+
+type Document with 
+    /// returns this DocumentOrShadow adopted stylesheets or sets them.
+    /// https://wicg.github.io/construct-stylesheets/#using-constructed-stylesheets
+    [<Emit("$0.adoptedStyleSheets{{=$1}}")>]
+    member __.adoptedStyleSheets with get(): CSSStyleSheet array = jsNative and set(v: CSSStyleSheet array) = jsNative
```

Comment:
I couldn't find a type that is shared among document and Element but both document, element and shadowroot should have adoptedStyleSheets available

Reply:
It should be ok to add it to both interfaces as we don't have interface mixins in F#. There are some `Document` extensions below in this file, so it'd be nice to put them together. There's no documentation about `adoptedStyleSheets`, I could only find [this](https://wicg.github.io/construct-stylesheets/#using-constructed-stylesheets) which says `DocumentOrShadowRoot`. Should this be a extension for ShadowRoot instead of Element?

Reply:
I thought the same but it looks like even elements have shadowRoots https://stackblitz.com/edit/js-a27roq